cmake/modules/FindJsonSchemaBuilder.cmake

```diff
@@ -10,8 +10,13 @@
 if(NOT TARGET JsonSchemaBuilder::JsonSchemaBuilder)
   if(KODI_DEPENDSBUILD OR CMAKE_CROSSCOMPILING)
     add_executable(JsonSchemaBuilder::JsonSchemaBuilder IMPORTED GLOBAL)
-    set_target_properties(JsonSchemaBuilder::JsonSchemaBuilder PROPERTIES
-                                                               IMPORTED_LOCATION "${NATIVEPREFIX}/bin/JsonSchemaBuilder")
+    if(CORE_SYSTEM_NAME STREQUAL windows OR CORE_SYSTEM_NAME STREQUAL windowsstore)
```

Comment:
I assume windowsstore is going to be defined somewhere in a later PR?
In that case this commit should probably be moved to that future PR.
Else, please remove it.